one-ui/src/test/resources/application-test.properties

```diff
@@ -1,3 +1,9 @@
 test.mlHost=localhost
-test.dataHubDeveloperUsername=data-hub-developer-user
-test.dataHubDeveloperPassword=data-hub-developer-user
\ No newline at end of file
+test.dataHubDeveloperUsername=test-data-hub-developer
```

Comment:
Are these still needed here? I doubt we need properties for them - I can't think of a reason why we'd change them in between test environments. 

If Installer is the class that is creating test users, I'd just define their username/passwords as static values on the Installer class.

Reply:
Since there is not going to be dependency on marklogic-data-hub:test, we will need this.